src/StreamJsonRpc/JsonRpc.cs

```diff
@@ -1102,6 +1111,21 @@ protected async Task<TResult> InvokeCoreAsync<TResult>(long? id, string targetNa
                 request.Arguments = arguments ?? EmptyObjectArray;
             }
 
+            return await this.InvokeCoreAsync<TResult>(request, cancellationToken).ConfigureAwait(false);
+        }
+
+        protected async Task<TResult> InvokeCoreAsync<TResult>(JsonRpcRequest request, CancellationToken cancellationToken)
```

Comment:
Does this overload need to be `protected` or can it be `private`?

Reply:
It can probably be private.  I originally had a subclass of JsonRpc, but found that was not necessary.